src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -38,6 +38,7 @@ package object csv {
         parserLib: String = "COMMONS",
         ignoreLeadingWhiteSpace: Boolean = false,
         ignoreTrailingWhiteSpace: Boolean = false,
+        treatEmptyValuesAsNulls: Boolean = false,
```

Comment:
The MiMa check failed because the addition of a new argument to this method changes its Java method signature and also risks source incompatibility.

@falaki, for this reason I think that, in retrospect, it would have been better to go with a builder pattern for this instead of a big Scala method with defaults arguments.